clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/security/OpenstackNamedAccountCredentials.groovy

```diff
@@ -77,6 +77,15 @@ class OpenstackNamedAccountCredentials implements AccountCredentials<OpenstackCr
     this.credentials = buildCredentials()
   }
 
+  /*
+   * Technically, just the name property providers this getter, but ran into an issue with mocking this class
```

Comment:
I removed this and skipped testing the bit that needed this. Will readdress once that has been refactored into the caching layer.